### DISCUSSION TITLE
Hide editing window

### DISCUSSION THREAD

**Discussion by MartinOrell** [GhostManager/Ghostwriter#483] [Q&A]
Is it possible to hide the editing window to only look at the preview without half the screen being covered by the editing window?

**Comment by chrismaddalena** [comment]
Hey @MartinOrell, I missed this in the Discussion topic. I can't think of a spot to view a preview alongside editing like that. Could you tell me more?

Or were you looking for the KDE MarkDown editor named _ghostwriter_?

**Reply by MartinOrell** [reply]
I'm talking about ghostwriter, yes.

I can have the editing window and preview window open at the same time.
![Untitled](https://github.com/user-attachments/assets/139446e6-806e-4d31-98bb-4f76014bfc23)

And I can have only the editing window open
![Untitled2](https://github.com/user-attachments/assets/cc1d2fed-24b6-4c82-976d-bf5c88607edf)

But I don't see a way to only look at the preview window. Sometimes I just want to look at the preview without editing.
